### PR TITLE
ignore error when calling resume() on streams

### PR DESCRIPTION
**Pull request by acarrasco**

### CODE REVIEW

File: tasks/run.js [hunk 1]
```diff
@@ -148,8 +148,12 @@ function makeTask(grunt) {
 
 
     // ensure that the streams are draining if we aren't already draining them (like quiet=true)
-    proc.stdout.resume();
-    proc.stdout.resume();
```
Comment (acarrasco) [inline tasks/run.js]: also, I think this was a typo and was meant to be `stderr`

**Comment by spalger** [comment]
LGTM

**Comment by acarrasco** [comment]
awesome, thanks! :)

**Comment by spalger** [comment]
Published with v0.5.1

**Pull request merged**